deployments/k8s/helm/signalfx-agent/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       annotations:
         {{- toYaml .Values.podAnnotations | trim | nindent 8 }}
     spec:
-      dnsPolicy: ClusterFirstWithHostNet
+      dnsPolicy: {{ .Values.dnsPolicy }}
```

Comment:
Do this same thing for the DaemonSet too so it is consistent.

Reply:
sorry the delay, I changed it to use the value

Reply:
Thanks! @keitwb  mind taking another look?

Reply:
I don't  see the change in the daemonset.

Reply:
@cohandv  ^^